### DISCUSSION TITLE
env variable in windows [May 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2075] [Discord Threads]
> 🙋‍♂️ **Gurukiran**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### probably you will have to use something like cross-env if you are on windows  
#  
> 💁 **o1lab**  
#### happy to accept a PR on it 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/971318124001513513)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
